### PR TITLE
Add a section about SPRAL in the README.md

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,3 +274,19 @@ model = Model(Ipopt.Optimizer)
 set_attribute(model, "pardisolib", "/full/path/to/libpardiso")
 set_attribute(model, "linear_solver", "pardiso")
 ```
+
+### SPRAL
+
+If you use Ipopt.jl with Julia ≥ v1.9, the linear solver [SPRAL](https://github.com/ralna/spral) is available.
+You can use it setting the `linear_solver` attribute:
```
Comment (amontoison) [inline README.md:281]:
```suggestion
You can use it by setting the `linear_solver` attribute:
```

File: README.md [hunk 2]
```diff
@@ -274,3 +274,19 @@ model = Model(Ipopt.Optimizer)
...
+```julia
+using JuMP, Ipopt
+model = Model(Ipopt.Optimizer)
+set_attribute(model, "linear_solver", "spral")
+```
+Note that the following environment variables must be set if you use SPRAL:
```
Comment (odow) [inline README.md:287]: before starting Julia? Or can I set them in Julia with `ENV[...] =  ...` before loading Ipopt?

If the latter, it'd be nice to add a test for this.

Comment (amontoison) [reply README.md:287]: It should work if we can set them with `ENV[...] = ...` before the `import Ipopt_jll`.

Comment (amontoison) [reply README.md:287]: I tested this following code and it's not working:
```julia
ENV["OMP_CANCELLATION"] = "TRUE"
ENV["OMP_NESTED"] = "TRUE"
ENV["OMP_PROC_BIND"] = "TRUE"

using Ipopt, JuMP

# An optimization problem
model = Model(Ipopt.Optimizer)
@variable(model, x)
@objective(model, Min, (x - 2)^2)

# Use the linear solver SPRAL
set_attribute(model, "linear_solver", "spral")
optimize!(model)
```
I need to start Julia with the environment variables already defined:
```
OMP_CANCELLATION=TRUE OMP_NESTED=TRUE OMP_PROC_BIND=TRUE julia
```

File: README.md [hunk 3]
```diff
@@ -274,3 +274,19 @@ model = Model(Ipopt.Optimizer)
...
+### SPRAL
+
+If you use Ipopt.jl with Julia ≥ v1.9, the linear solver [SPRAL](https://github.com/ralna/spral) is available.
+You can use it by setting the `linear_solver` attribute:
+```julia
+using JuMP, Ipopt
+model = Model(Ipopt.Optimizer)
+set_attribute(model, "linear_solver", "spral")
+```
+Note that the following environment variables must be set if you use SPRAL:
```
Comment (odow) [inline README.md:287]:
```suggestion
Note that the following environment variables must be set before starting Julia:
```

**Pull request merged**